src/main/java/frc/trigon/robot/motorsimulation/SingleJointedArmSimulation.java

```diff
@@ -0,0 +1,36 @@
+package frc.trigon.robot.motorsimulation;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import frc.trigon.robot.utilities.Conversions;
+
+public class SingleJointedArmSimulation extends MotorSimulation {
+    private final SingleJointedArmSim armSim;
+
+    public SingleJointedArmSimulation(DCMotor motor, double gearRatio, double momentOfInertia, double retractedArmLengthMeters, double minimumAngleRadians, double maximumAngleRadians, boolean simulateGravity) {
+        armSim = new SingleJointedArmSim(motor, gearRatio, momentOfInertia, retractedArmLengthMeters, minimumAngleRadians, maximumAngleRadians, simulateGravity, minimumAngleRadians);
+    }
```

Comment:
Don't have an input for the moi, estimate it with the weight using the SingleJointedArmSim.estimateMOI() function